src/worker.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var queue = require('queue-async');
+var map = require(process.argv[2]);
+
+var q = queue();
+var sources = [];
+
+JSON.parse(process.argv[3]).forEach(function(source) {
+  q.defer(loadSource, source);
+});
+
+function loadSource(source, done) {
+  if (source.mbtiles) sources.push({name: source.name, getTile: require('./mbtiles')(source, done)});
```

Comment:
Are we waiting for the `ready` callback to be called anywhere? For the `remote` parser it appears that the getTile parser is not actually returned by the time the `ready` callback fires. I am not sure if this is a bug, but the flow is a bit difficult to follow.